pkg/auth/token.go

```diff
@@ -40,7 +40,8 @@ func NewAuthenticationTokenWithParams(params map[string]string) (Provider, error
 	}
 }
 
-// NewAuthenticationToken return a interface of Provider with a string token.
+// Return a token auth provider that will use the specified token to
+// talk with Pulsar brokers
```

Comment:
Externally exposed functions (aka, capitalized) need to start with the function name when adding a comment.  e.g: 

```
NewAuthenticationToken return a token ....
```

Otherwise these comments will not be displayed in `godoc`

Reply:
Got it, I didn't know it was expecting that. Will fix all of them.